transport/tlscommon/testdata/rsa_1024.key.pem

```diff
@@ -0,0 +1,16 @@
+-----BEGIN PRIVATE KEY-----
+MIICdgIBADANBgkqhkiG9w0BAQEFAASCAmAwggJcAgEAAoGBAKPO/jzZ6W7voTZI
+z4oC1SkppQ0rXdDqnDplJbHoBJTBJAgK15Evpw5A/qeqOkIBmeuZnr5myfQMU1eL
+R0GfGBOveQYFWvzIsc1jWtgo2buDmAlyXdavC375BMV46fPkVVrgIfhWOW2WBiJc
+Nsi0hxm6xOZNxfGJX6C+gA5AH+N5AgMBAAECgYAD5fw6Zyge6Aeu4FGSTy7mC3WM
+ydv+8DLR99nRx6V1qeyK3yfIiHxDn4CbLYoOJTyPZRqOtuj5iVvbTO3GbIwg09tW
+4MMyS2AABIaO8Ke2MdXseI1Dt9sY7TnAPs8tz6KOEPksWXYOroqrzqXXmlG/yEei
+Fk9Z/UZB3ue33oq+IQJBAMm53Bi7ck2A7O4ueaeX9SkC5XpFpaUAimY5h8m+J0aB
+vXUjEX0BzEj7/+ocX+KaEXE0gfvQZHl2PUY5qCwnbdkCQQDP4YUknD+1lFH7l8tJ
+1whZfPEKn7MYAAS9wI5q11CTeaSkvq3z+5gX0EwLBn7WSqfcR3vQOBmyz0uzZhws
+e36hAkAP+Z4Kf12v8ZPR0PBla01I8CfIJRfXF1HegpPUUDDADqo4Soyp/6hz5zD/
+Ezwsr9LNykC49mnejJSRqSM+S+kRAkEAhKEZBmueBia0S7XkIJ9OF3Isg5+ybwyL
++dihxK7NHNpOXkG90F1kA0WFTr99KxGEmXkOGKHCW6AAZ1wte3/rIQJASAjYaX8z
+cvqU4hUIsh0A1fQDD4j1HYrkOdsThrAoWRPu2mBDsD6IWnVmHzRB8coTZllP+mBA
+JL2QoEeSSdfniw==
+-----END PRIVATE KEY-----
```

Comment:
checking a private key into the repo is going to trigger an alert about private keys.  So at a minimum we would need to make sure this gets added to the "accepted" list.   But given that and the fact that the cert is expired, maybe we should generate the cert each time.  Thoughts?

Reply:
I think moving to generate the certs is a good idea.
It would need to run as a separate step, not as part of the test case; we want the test cases to run with whatever fips checks there are and not fail in startup

Reply:
Can't we just Base64-encode these since they're merely test dummies, do we really need to keep the keys in plain text files?

Reply:
I actually started out with trying to generate the certs. The problem is that in FIPS mode, we can't even generate the cert with key length < 2048 bits.  That's why I had to manually generate them first outside of FIPS mode and check-in the files this way.

I will try @michel-laterman's suggestion of generating the certs outside the test case (which runs in FIPS mode).  If for some reason that doesn't do it, I'll try @mauri870's suggestion of base64-encoding the keys and decoding them in the test case just before using them.